src/test/run-pass/rvalue-static-promotion.rs

```diff
@@ -0,0 +1,17 @@
+// Copyright 2017 The Rust Project Developers. See the COPYRIGHT
+// file at the top-level directory of this distribution and at
+// http://rust-lang.org/COPYRIGHT.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![feature(rvalue_static_promotion)]
```

Comment:
(I believe it's not covered in `src/test/COMPILER_TEST.md`, though that doc does talk about "revisions" which seems like that's what you would want, but other tests I've randomly peeked into don't seem to use that, so I see no reason for the feature to be toggled when they run)